### ISSUE TITLE
Feeder and Felaqua support

### ISSUE THREAD

**Issue by wifispray** [benleb/surepetcare#30]
Hello

I couldn’t tell from the read me if these devices are available?

also is there a hacs install?

**Comment by Daneish** [comment]
Yes to both your questions. It can be found under SureHA in HACS.
